commerce_coordinator/apps/commercetools/views.py

```diff
@@ -82,3 +90,51 @@ def post(self, request):
         send_order_confirmation_email(lms_user_id, customer.email, canvas_entry_properties)
 
         return Response(status=status.HTTP_200_OK)
+
+
+class OrderSanctionedView(APIView):
+    """View to sanction an order and deactivate the lms user"""
+    permission_classes = [IsAdminUser]
+
+    # authentication_classes = [] TODO: Update this with OAuth authentication
```

Comment:
No need to explicitly specify authentication_classes in the endpoint as we have `JwtAuthentication` set as default in [REST_FRAMEWORK](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/settings/base.py#L277)